tests/test_tmjob.py

```diff
@@ -10,6 +10,7 @@
 from pytom_tm.tmjob import TMJob, TMJobError
 from pytom_tm.io import read_mrc, write_mrc, UnequalSpacingError
 from pytom_tm.extract import extract_particles
+from test_weights import CTF_PARAMS, ACCUMULATED_DOSE, TILT_ANGLES
```

Comment:
this is fine, often for bigger programming suites we have a dedicated `common.py` file inside the test directory to define these parameters and import the from there.
Another option is a default `class` in this common file that sets all these options on initialization and all the other tests inherit from that one